playbooks/provisioning/openstack/README.md

```diff
@@ -535,13 +537,31 @@ This example runs against app nodes. The list of options include:
   - masters
   - infra_hosts
 
+#### Attaching additional RHN pools
+
+```
+---
+- hosts: cluster_hosts
+  tasks:
+  - name: Attach additional RHN pool
+    become: true
+    command: "/usr/bin/subscription-manager attach --pool=<pool ID>"
+    register: attach_rhn_pool_result
+    until: attach_rhn_pool_result.rc == 0
+    retries: 10
+    delay: 1
```

Comment:
This delay may be too low for subscription-manager related issues.

Reply:
what delay do you propose then?

Reply:
afaik the cdn.redhat.com DNS record  TTL is 20 seconds. Maybe 10 or 15 seconds could be fine.

Reply:
I see. I just used the same values as in the rest of `subscription-manager` tasks. I guess those should be adjusted as well then?

Reply:
@ioggstream this is a delay between retries not a timeout. If the task takes 30 seconds to succeed, this won't kill it.

It's just that if it fails, we will wait 1 second and retry. And do that 10 times.